internal/compiler/emitter.go

```diff
@@ -258,7 +399,12 @@ func sourceFileMayBeEmitted(sourceFile *ast.SourceFile, host EmitHost, forceDtsE
 		return false
 	}
 
-	// !!! Source file from node_modules are not emitted
+	// !!! Source file from node_modules are not emitted. In Strada, this depends on module resolution and uses
+	// `sourceFilesFoundSearchingNodeModules` in `createProgram`. For now, we will just check for `/node_modules/` in
+	// the file name.
+	if strings.Contains(sourceFile.FileName(), "/node_modules/") {
```

Comment:
I would suggest using `.Pah()` here but I assume this is just temporary?

Reply:
Yes, this is temporary.